ldap_validator/library/utils/ldap_messages.py

```diff
@@ -0,0 +1,27 @@
+####################################################################
+# ### ldap_messages.py                                         ###
+####################################################################
+# ### Author: SAS Institute Inc.                                 ###
+####################################################################
+#                                                                ###
+# Copyright (c) 2020, SAS Institute Inc., Cary, NC, USA.         ###
+# All Rights Reserved.                                           ###
+# SPDX-License-Identifier: Apache-2.0                            ###
+#                                                                ###
+####################################################################
+# Error Messages written to console and logs
+#
+# Error Messages
+
+OPTION_ERROR = "ERROR: option {} not recognized"
+OPTION_VALUES_ERROR = "ERROR: Provide valid values for all required options. Check options -i, -p and -H."
```

Comment:
Options "-i, -p, -H" ??     what options are these??

Reply:
Those are required options for pre-install-report that were copied over and never changed to match required option (-s) for ldap-validator. The change/fix will be incorporated in my next push.